postgres-persistence/src/main/java/com/netflix/conductor/postgres/PostgresDataSourceProvider.java

```diff
@@ -90,6 +92,9 @@ private void flywayMigrate(DataSource dataSource) {
 
         flyway.setLocations(Paths.get("db","migration_postgres").toString());
         flyway.setDataSource(dataSource);
+        flyway.setBaselineOnMigrate(true);
+        flyway.setBaselineVersion(MigrationVersion.fromVersion("0"));
+        flyway.setTable("schema_version");
```

Comment:
Please refrain form hardcoding the table name here.

Reply:
I agree, we use multiple Flyway migrations in the same schema and the version table must be customizable. I would add that `schema_version` is just a Flyway default, I have typically recommended to _always_ use something more descriptive like `${APP_NAME}_schema_version, i.e. `conductor_schema_version`.

Reply:
> Please refrain form hardcoding the table name here.

Updated, remove the hardcoding name.